core/application/models/dbModels/NotificationModel.ts

```diff
@@ -53,8 +38,8 @@ export default class NotificationModel implements NosqlModel<NotificationFields>
     return {
       ...remainingNotificationFields,
       userId,
-      type: parts[1] as NotificationType,
-      id: parts[2]
+      type: parts[0] as NotificationType,
+      id: parts[1]
```

Comment:
What if parts is an empty array?

Reply:
Updated in 49f319a8110867cd9a5be505206104734474bfa3